### ISSUE TITLE
Ore tracking not working.

### ISSUE THREAD

**Issue by dreamwraith** [aaron-yang0327-development/ValheimMods#1]
Cannot seem to get this to work for mineral deposits. Opening track dialogue while looking at copper ore for example brings up empty dialogue. Trying to create manually using "Copper" for the ID name and unchecking exact match also does not seem to work for auto marking

**Comment by dreamwraith** [comment]
Sorry, I should have added, this is AFTER it has been hit with a pickaxe.

**Comment by aaron-yang0327-development** [comment]
**tl;dr; down the comment**

Hello, thanks for the report, I just looked into it and you're right, I checked the object information of a "hit" copper deposit and it doesn't actually use the type "MineRock" but the type "MineRock5" instead, in simpler terms the deposit is not a MineRock but a MineRock5 object

My mod detects these types and checks for the object's name, (hence why you see (Clone) most of the time in the UI's Object ID, it's because the objects are clones of a database from Valheim).

So yes it is indeed a bug that I overlooked, but if I added MineRock5(hit deposits) to be detectable, it would be meaningless because 

1. after a boulder has been hit, it will transform into a similar looking one but with pieces of predefined chunks that you can chip away, each of those chips have different slightly similar names (easily solvable by not enabling exact match)
2. and the important reason, you won't be able to differentiate between other boulders, all of those pieces are ambiguous, meaning, Rock, Copper, Iron, etc. 's pieces are all the same.

I could find a way to solve this, but would most likely be a low priority as tracking a copper deposit pre-struck works no problem and that I'm working on the new colored pin feature for each tracked object for added customization, I was too immersed hence why I just read this now, sorry.

tl;dr; easy fix, technically, but that would mean the mod can't differentiate a rock, copper, or iron boulder post-struck with pickaxe, time consuming fix, to be able to differentiate other objects. Would set this low priority for now as I'm working on the colored pin feature and you can track a pre-struck copper instead. Sorry for the late reply as well, was busy with the mod and other stuff.

**Comment by dreamwraith** [comment]
No worries at all, thanks for the detailed explanation.

I'm a software developer by day (non game dev) so am super familiar with why things are more complex than they seem. I've been thinking about dabbling in modding for the server we run myself, but looking at code at night after looking at code all day always puts me off. So I appreciate the open work you and others are doing!

**Comment by aaron-yang0327-development** [comment]
thank you! hope you have a great time with my first mod.
